insights/parsers/mysqladmin.py

```diff
@@ -15,6 +15,38 @@
 from insights.specs import Specs
 
 
+@parser(Specs.mysqladmin_status)
+class MysqladminStatus(LegacyItemAccess, CommandParser):
+    """
+    Module for parsing the output of the ``mysqladmin status`` command.
+    Typical output looks like::
+        Uptime: 1103965 Threads: 1820 Questions: 44778091 Slow queries: 0 Opens: 1919 Flush tables: 1 Open tables: 592 Queries per second avg: 40.561
+    Example:
+        >>> "Uptime" in mysqlstat
+        True
+        >>> mysqlstat['Threads']
+        '1820'
+        >>> mysqlstat['Flush_tables'] == '1'
+        True
+    """
+    def parse_content(self, content):
+        if not content:
+            raise ParseException("Input content is empty.")
+        self.data = {}
+
+        if len(content) == 1:
+            st = list(content)[0].strip()
```

Comment:
`content` is already a list, not sure why you are casting it here?

Reply:
casting it to string to use the replace method

Reply:
@asyedham `content` is already a `list`.  When you cast `list(content)` you get exactly the same list:

```
In [1]: content = ['a and b and c']                                                                                                                                                                                                           

In [2]: content                                                                                                                                                                                                                               
Out[2]: ['a and b and c']

In [3]: list(content)                                                                                                                                                                                                                         
Out[3]: ['a and b and c']

In [4]: list(content)[0]                                                                                                                                                                                                                      
Out[4]: 'a and b and c'
```

Reply:
If you'll fix this issue with the casting then I'll approve your other code as-is.

Reply:
> @asyedham `content` is already a `list`. When you cast `list(content)` you get exactly the same list:
> 
> ```
> In [1]: content = ['a and b and c']                                                                                                                                                                                                           
> 
> In [2]: content                                                                                                                                                                                                                               
> Out[2]: ['a and b and c']
> 
> In [3]: list(content)                                                                                                                                                                                                                         
> Out[3]: ['a and b and c']
> 
> In [4]: list(content)[0]                                                                                                                                                                                                                      
> Out[4]: 'a and b and c'
> ```

Got it!

Reply:
> If you'll fix this issue with the casting then I'll approve your other code as-is.

Done